src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__stars {
+    background-image: url(../../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--4 :nth-child(n+5) {
```

Comment:
What about other modifiers? Would be great to implement all of them, so you can easily switch between amount of active stars